src/components/Search/__tests__/Search.test.tsx

```diff
@@ -0,0 +1,140 @@
+import React from 'react';
+import {fireEvent, render, screen} from '@testing-library/react';
+import {TeamsType, UserDataType} from 'types/types';
+import Search from '../Search';
+
+describe('Search', () => {
```

Comment:
Created 3 tests

- Testing with the mock of `TeamsType[]`, if the filtering works
- Testing with the mock of `UserDataType[]`, if the filtering works
- Testing if the error is updated if the filtered array returns empty